README.md

```diff
@@ -102,13 +114,16 @@ https://github.com/aiven/klaw-cluster-api
 
 ## Install
 
-mvn clean install
+1. Clone this repo with submodules: `git clone --recursive git@github.com:aiven/klaw.git`
+2. Run `make` to install dependencies and setup both Klaw and the Klaw-Cluster-API
+3. [Optional] edit any configs using `make edit-config` for Klaw or `make edit-api-config` for Klaw-Cluster-API
+4. To run, you can use `make run` and `make run-api` in different terminal windows or you can run `make -j2 run run-api` to execute both (NOTE: this will mix output and make debugging harder)
```

Comment:
I think using `make` in the example is quite neat but it's not what Java projects usually have in their install guides. I think we should keep these instructions as they are quite clean and portable, but also mention the `mvn` and `java` commands.
Maybe 2 sections under this, "with make" and "manual"

Reply:
Makes sense, multiple documented options is almost always a good thing :)

Reply:
Agree, and in this case is the combination of "here, you have the handy way" and "this is the one you were expecting to see".